common/app/common/dfp/DfpData.scala

```diff
@@ -230,19 +222,21 @@ case class GuLineItem(
     orderId: Long,
     name: String,
     lineItemType: GuLineItemType,
-    startTime: DateTime,
-    endTime: Option[DateTime],
+    startTime: LocalDateTime,
+    endTime: Option[LocalDateTime],
     isPageSkin: Boolean,
     sponsor: Option[String],
     status: String,
     costType: String,
     creativePlaceholders: Seq[GuCreativePlaceholder],
     targeting: GuTargeting,
-    lastModified: DateTime,
+    lastModified: LocalDateTime,
 ) {
 
-  val isCurrent = startTime.isBeforeNow && (endTime.isEmpty || endTime.exists(_.isAfterNow))
-  val isExpired = endTime.exists(_.isBeforeNow)
+  val now = LocalDateTime.now()
+
+  val isCurrent = startTime.isBefore(now) && (endTime.isEmpty || endTime.exists(_.isAfter(now)))
+  val isExpired = endTime.exists(_.isBefore(LocalDateTime.now()))
```

Comment:
switch out `LocalDateTime.now()` for `now` in this line too?

Reply:
Done